README.md

```diff
@@ -1,393 +1,296 @@
 
-Run `test.html` in the `/var/www/app/` folder using Safari:
 
-> `testee test.html --root /var/www/app/ --browsers safari`
+## Installation
 
-Run the online [Underscore](http://underscorejs.org/) QUnit tests in Phantom and Firefox and output code coverage statistics:
+[Node.js](http://nodejs.org/) `>= 4` is required. To install, type this at the command line:
+```shell
+npm install testee
+```
 
-> `testee test/index.html --root http://underscorejs.org --browsers phantom,firefox --coverage`
+You can choose to install globally with `-g` if you wish, but it is recommended that you install per-project so that CI environments and all members of your team have instant access to the same dependencies.
 
-Run `tests/qunit.html` with PhantomJS from the current folder, use port `8080` instead of `3996` and
-use the `Spec` reporter which prints more detailed test results:
 
-> `testee tests/qunit.html --port 8080 --reporter Spec`
+## Command Line Usage
 
-Run `tests/mocha.html` using `testee.json` as the configuration file (see the [configuration API](#configuration-api)
-for how this file should look like):
+See available options:
+```shell
+testee --help
+```
 
-> `testee tests/mocha.html -c testee.json`
+Test with one or multiple files:
+```shell
+testee test.html
+```
+```shell
+testee test1.html test2.html
+```
 
-Run `tests/jasmine.html` using [Google Chrome Canary](https://www.google.com/intl/en/chrome/browser/canary.html):
+Save keystrokes with a base path/URL:
+```shell
+testee test1.html test2.html --root=/var/www/app/
+```
+```shell
+testee test1.html test2.html --root=http://yourapp/
+```
 
-> `testee tests/jasmine.html --browsers canary`
+Specify browsers:
+```shell
+testee test.html --browsers=firefox,safari
+```
 
+Use a [configuration file](#configuration-api):
+```shell
+testee test.html --config=testee.json
+```
+```shell
+testee test.html --config=testee.js
+```
 
-### CI integration
 
-Because Testee allows to use different reporters for the test result output it is easy to obtain XUnit
-style XML files that integrate with CI servers like [Jenkins](http://jenkins-ci.org/). Just use the `XUnit`
-reporter and write the output into a file. The following example runs `tests/qunit.html` in Firefox and writes
-the result XML into `testresults.xml`:
+## Programmatic Usage
 
-> `testee test/index.html --browsers firefox --reporter XUnit > testresults.xml`
+For custom scripts (including [gulp](https://npmjs.com/gulp)):
 
-You can get more information about the available reporters in the [Reporters](#reporters) section.
+```js
+const testee = require('testee');
 
+const browsers = 'phantom';
+const config = { reporter: 'spec' };
+const files = ['test/test.html'];
 
-### Capturing `console.log` and `console.error`
+testee.test(files, browsers, config)
+  .then(() => console.log('done!'));
```

Comment:
The readme will need to be updated next year when Node 4's support ends. I would hope that developers using Node 4 are aware of ES2015/ES6 features. Still want this changed?